src/java.base/share/classes/java/lang/classfile/constantpool/ConstantValueEntry.java

```diff
@@ -42,6 +42,8 @@ public sealed interface ConstantValueEntry extends LoadableConstantEntry
     /**
      * {@return the constant value}  The constant value will be an {@link Integer},
      * {@link Long}, {@link Float}, {@link Double}, or {@link String}.
+     *
+     * @see java.lang.classfile.Attributes#constantValue()
```

Comment:
I wish you can:
1. Add a link on the `{@code ConstantValue}` in class main javadoc so it's like `{@link ConstantValueAttribute ConstantValue}` or `{@link Attributes#constantValue ConstantValue}`
2. Change the `@see` link to `ConstantPoolBuilder#constantValueEntry` as that pairs up with this method.

```suggestion
     * @see ConstantPoolBuilder#constantValueEntry(ClassDesc)
```